### ISSUE TITLE
Terminal output timestamp does not meet color contrast ratio

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52570]
**Environment Details:** 
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated: MAS1.3.1
ColorContrast,MAS1.4.3

**Repro Steps:**
1. Launch VS Code.
2. Go to View menu and Select Output.
3. Select Log(Main) or any of the items from the combobox after Terminal tab item.

**Actual:**
Color contrast ratio is less than 4.5:1 for the code coming after selecting list item from drop down.

**Expected:**
Color Contrast ratio should more than 4.5:1.
Issue observed on all color themes.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not be able to know that the editor is read only.

**MAS Reference**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}](url)

**Attachment for Reference:** 
![accessibility a11y_vscode_viewoutput_colorcontrast](https://user-images.githubusercontent.com/39204909/41708575-92c17fdc-755e-11e8-8218-250c2708e504.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by misolori** [comment]
This passes on our light theme but fails on dark
